openff/bespokefit/executor/services/coordinator/stages.py

```diff
@@ -144,7 +145,8 @@ async def _update(self):
                 f"http://127.0.0.1:"
                 f"{settings.BEFLOW_GATEWAY_PORT}"
                 f"{settings.BEFLOW_API_V1_STR}/"
-                f"{settings.BEFLOW_FRAGMENTER_PREFIX}/{self.id}"
+                f"{settings.BEFLOW_FRAGMENTER_PREFIX}/{self.id}",
+                headers={"bespokefit-token": settings.BEFLOW_API_TOKEN},
```

Comment:
(I'm not in any way familiar with internet security, just wanted to raise this because it seemed smelly) Is it safe to transmit API tokens using vanilla `http` or is this a use case for `https`?

Reply:
Ahh good catch this is where the server is connecting to itself but yes probably still good to use `https`.

Reply:
But for some reason this breaks the server 
```
"SSLError: [SSL] record layer failure (_ssl.c:1006)"
```